src/frontend/komponenter/Fagsak/Behandlingsresultat/Oppsummeringsboks.tsx

```diff
@@ -25,39 +26,26 @@ import {
     sorterUtbetaling,
 } from '../../../utils/formatter';
 
-const TableHeaderAlignedRight = styled.th`
-    text-align: right;
-`;
-
-const TableDataAlignedRight = styled.td`
-    text-align: right;
-`;
-
 const AlertAlignedRight = styled(Alert)`
     float: right;
 `;
 
-const FlexDiv = styled.div`
-    display: flex;
-`;
-
-const UtbetalingsbeløpTable = styled.table`
-    width: 100%;
-    padding-bottom: 1rem;
-`;
-
-const VenstreTekst = styled(BodyShort)`
-    text-align: left;
-    font-weight: bold;
-    width: 50%;
-    margin: 1.25rem 0;
+const UtbetalingsbeløpStack = styled(VStack)`
+    padding: ${ASpacing6} ${ASpacing10} ${ASpacing4} 0;
 `;
 
-const HøyreTekst = styled(BodyShort)`
-    text-align: right;
-    font-weight: bold;
-    width: 50%;
-    margin: 1.25rem 2.5rem 1.25rem 0;
+interface UtbetalingsbeløpRadProps {
```

Comment:
Kan bruke PropsWithChildren fra React i stedet for eget interface

Reply:
Vi er ikke på react 18 enda, og da funker det ikke på samme måte, så droppet å gjøre det her